pkg/epinio/components/application/AppSource.vue

```diff
@@ -349,6 +349,7 @@ export default Vue.extend<Data, any, any, any>({
             :label="t('epinio.applications.steps.source.archive.file.button')"
             :mode="mode"
             :raw-data="true"
+            :accept="'.zip, .tar, .tgz, .tbz, .txz'"
```

Comment:
Can we confirm these cover the same file extensions that the backend does? https://github.com/epinio/helm-charts/blob/3954c214de3d7b957cfc2054ba4fa4bfa140f5a3/chart/epinio/templates/stage-scripts.yaml#L27-L62 @jimmykarily

Reply:
It looks like the tar compression extensions aren't quite correct given the comment at the top of that link. What happens when you provide a `tar.gz` file?

Reply:
I double checked and it seems not working with nested extensions 😢 

![Screenshot 2022-08-05 at 11 15 53](https://user-images.githubusercontent.com/5009481/183045543-c28fbba5-c2c8-49de-a376-9b4320323af9.png)

Reply:
Also it seems like you'll have to use ` .gz, .bz2, .xz`

Reply:
For reference, this is the list of content types we check against in the "upload" endpoint: https://github.com/epinio/epinio/blob/83b68f9510a3f1681eec0ef558a9f26188e14087/internal/api/v1/application/upload.go#L22-L28

[We read the first 512 bytes of the file](https://github.com/epinio/epinio/blob/83b68f9510a3f1681eec0ef558a9f26188e14087/internal/api/v1/application/upload.go#L90) to get the type but that's obviously not possible at the front end.